doc/design/milestone-0.0.7/example.describe.yaml

```diff
@@ -0,0 +1,130 @@
+replaces: prometheusoperator.0.15.0
```

Comment:
What does the `replaces` stand for?

Reply:
There *should* be a description of this yaml object in the [CSV operator metadata section](https://github.com/operator-framework/operator-lifecycle-manager/blob/16ff8f983b50503c4d8b8015bd0c14b5c7d6786a/Documentation/design/building-your-csv.md#operator-metadata), but I can't find any mention. `replaces` informs the OLM that this CSV should replace an already-loaded CSV, i.e. update CSV versions, in the cluster.

Reply:
Thanks, got it, maybe we can add that to the docs when it's clearer.